packages/dd-trace/test/encode.spec.js

```diff
@@ -3,6 +3,7 @@
 const msgpack = require('msgpack-lite')
 const codec = msgpack.createCodec({ int64: true })
 const id = require('../src/id')
+const { Int64BE } = require('int64-buffer') // TODO: remove dependency
```

Comment:
Why is this still needed anyway without `msgpack-lite`?

Reply:
I'm not sure we do, but I wasn't trying to remove it with these changes since it didn't appear to be a bottleneck.